pkg/api/v20210901preview/openshiftcluster.go

```diff
@@ -97,6 +97,9 @@ type ClusterProfile struct {
 
 	// The ID of the cluster resource group.
 	ResourceGroupID string `json:"resourceGroupId,omitempty"`
+
+	// If FIPS validated crypto modules are used
+	FipsEnabled bool `json:"fipsEnabled,omitempty"`
```

Comment:
Same as Admin API - no bools.

Reply:
Overall MSFT does not like abbreviation on the API but this might fly... Need POC pr for this too :)

Reply:
I am not sure what to expect, but here goes nothing https://github.com/Azure/azure-rest-api-specs/pull/16028